src/assets/styles/pages/homepage.scss

```diff
@@ -0,0 +1,169 @@
+body {
+ width: 100%;
+ height: 100vh;
+ overflow: hidden;
+ box-sizing: border-box;
+ font-family: 'Poppins', sans-serif;
+}
+
+.header {
+  position: fixed;
+  inset: 0 0 auto 0;
```

Comment:
inset: 0; is enough it means top bottom right and left 0

Reply:
It doesn't work. I tried some combinations, and this one straightened it to the middle